cmd/ocm-backplane/script/describeScript.go

```diff
@@ -53,12 +54,21 @@ func newDescribeScriptCmd() *cobra.Command {
 			}
 			// ======== Initialize backplaneURL == ========
 			backplaneHost := urlFlag
-			if backplaneHost == "" {
+			if backplaneHost != "" {
+				// Validate urlFlag if it's provided
+				parsedURL, parseErr := url.ParseRequestURI(urlFlag)
```

Comment:
Any possibilities to get `backplaneHost` from `ocm get /api/clusters_mgmt/v1/environment` ? 
if not, pls enable only for FredRamp, we don't encourage users to set bp-url from the config.